sopel/config/core_section.py

```diff
@@ -178,6 +178,9 @@ def homedir(self):
     It is a regular expression (so the default, ``\.``, means commands start
     with a period), though using capturing groups will create problems."""
 
+    reply_errors = ValidatedAttribute('message_logs_sender',bool,default=False)
```

Comment:
What I meant is that the name should be consistent across the section - if the variable name is `reply_errors`, the ValidatedAttribute should use the same for the key.